src/server/system_services/stats_aggregator.js

```diff
@@ -420,7 +420,7 @@ async function _partial_buckets_info(req) {
 
     try {
         for (const bucket of system_store.data.buckets) {
-            if (String(bucket.system._id) !== String(req.system._id)) return;
+            if (String(bucket.system._id) !== String(req.system._id)) continue;
```

Comment:
Is this fix related to this PR?

Reply:
The fix is not related to the PR, I just did that since it crashed the partial metrics which means Prometheus metrics when we have namespace buckets.